util/pyclient/bft_client.py

```diff
@@ -96,7 +94,7 @@ def __init__(self, config, replicas, ro_replicas=[]):
         self.signing_key = None
         if txn_signing_key_path:
             with open(txn_signing_key_path, 'rb') as f:
-                self.signing_key = RSA.import_key(f.read())
+                self.signing_key = serialization.load_pem_private_key(f.read(), password=None)
```

Comment:
Is it okay to not allow clients to use RSA (thought that this was a runtime parameter of the replica)?